004-quarkus-HHH-and-HV/src/main/resources/import.sql

```diff
@@ -0,0 +1,12 @@
+truncate table item cascade ;
+truncate table customer cascade ;
+truncate table account_in_role cascade ;
+truncate table account cascade ;
+truncate table role cascade ;
+start TRANSACTION ;
```

Comment:
This is redundant IMO, because you have set `quarkus.hibernate-orm.database.generation=drop-and-create`
Plus it throwing errors anyway.

Reply:
Removed, thanks!